src/data/finset/pointwise.lean

```diff
@@ -977,6 +1000,9 @@ by { simp_rw ←coe_subset, push_cast, exact set.set_smul_subset_iff }
 @[to_additive] lemma subset_smul_finset_iff : s ⊆ a • t ↔ a⁻¹ • s ⊆ t :=
 by { simp_rw ←coe_subset, push_cast, exact set.subset_set_smul_iff }
 
+@[simp, to_additive] lemma card_smul_finset (a : α) (s : finset β) : (a • s).card = s.card :=
+card_image_of_injective _ $ mul_action.injective _
```

Comment:
These two are already in the `finset` namespace, aren't they?

Reply:
Yes, but there is both `(s : finset α) • (t : finset β)` called `smul`, and `(a : α) • (s : finset β)` called `smul_finset`. This naming convention isn't great, but it's thoroughly followed in the `pointwise` files.